HISTORY.rst

```diff
@@ -5,6 +5,8 @@ History
 0.9.x (2025-xx-xx)
 ------------------
 
+* Add train_conformalize_train_split()
```

Comment:
No need to update history for v1 code (way too many features, that's the purpose of the migration guide, and we'll write a special release note)